src/scripts/admin/filters/button-enhanced.js

```diff
@@ -1,7 +1,7 @@
 import classnames from 'classnames';
 import iconsObj from '../utils/button-icons';
 
-const { assign, merge } = require('lodash');
+import { assign, merge } from 'lodash';
 
 const { registerBlockStyle, unregisterBlockStyle } = wp.blocks;
 const { __ } = wp.i18n;
```

Comment:
this is not how i would import, if you weren't running through webpack, yes, but this should be the way you load it. 
```suggestion
const { __ } from '@wordpress/i18n';
```